Issue: `make coverage` doesn't work locally

### Issue you'd like to raise.

When I set up the local environment and try to run `make coverage`, I get this error:

```bash
; make coverage
poetry run pytest --cov \
		--cov-config=.coveragerc \
		--cov-report xml \
		--cov-report term-missing:skip-covered
================================================================================================================ test session starts ================================================================================================================
platform darwin -- Python 3.9.17, pytest-7.4.0, pluggy-1.2.0
rootdir: /Users/cjameson/workspace/cjcjameson/langchain/libs/langchain
configfile: pyproject.toml
plugins: asyncio-0.20.3, cov-4.1.0, vcr-1.0.2, syrupy-4.2.1, mock-3.11.1, anyio-3.7.1, dotenv-0.5.2, socket-0.6.0
asyncio: mode=strict
collected 2832 items / 1 error / 4 skipped                                                                                                                                                                                                          
INTERNALERROR> Traceback (most recent call last):
INTERNALERROR>   File "/Users/cjameson/workspace/cjcjameson/langchain/venv/lib/python3.9/site-packages/_pytest/config/__init__.py", line 1611, in getoption
INTERNALERROR>     val = getattr(self.option, name)
INTERNALERROR> AttributeError: 'Namespace' object has no attribute 'only_extended'
INTERNALERROR> 
INTERNALERROR> The above exception was the direct cause of the following exception:
INTERNALERROR> 
INTERNALERROR> Traceback (most recent call last):
INTERNALERROR>   File "/Users/cjameson/workspace/cjcjameson/langchain/venv/lib/python3.9/site-packages/_pytest/main.py", line 270, in wrap_session
INTERNALERROR>     session.exitstatus = doit(config, session) or 0
INTERNALERROR>   File "/Users/cjameson/workspace/cjcjameson/langchain/venv/lib/python3.9/site-packages/_pytest/main.py", line 323, in _main
INTERNALERROR>     config.hook.pytest_collection(session=session)
INTERNALERROR>   File "/Users/cjameson/workspace/cjcjameson/langchain/venv/lib/python3.9/site-packages/pluggy/_hooks.py", line 433, in __call__
INTERNALERROR>     return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)
INTERNALERROR>   File "/Users/cjameson/workspace/cjcjameson/langchain/venv/lib/python3.9/site-packages/pluggy/_manager.py", line 112, in _hookexec
INTERNALERROR>     return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
INTERNALERROR>   File "/Users/cjameson/workspace/cjcjameson/langchain/venv/lib/python3.9/site-packages/pluggy/_callers.py", line 155, in _multicall
INTERNALERROR>     return outcome.get_result()
INTERNALERROR>   File "/Users/cjameson/workspace/cjcjameson/langchain/venv/lib/python3.9/site-packages/pluggy/_result.py", line 108, in get_result
INTERNALERROR>     raise exc.with_traceback(exc.__traceback__)
INTERNALERROR>   File "/Users/cjameson/workspace/cjcjameson/langchain/venv/lib/python3.9/site-packages/pluggy/_callers.py", line 80, in _multicall
INTERNALERROR>     res = hook_impl.function(*args)
INTERNALERROR>   File "/Users/cjameson/workspace/cjcjameson/langchain/venv/lib/python3.9/site-packages/_pytest/main.py", line 334, in pytest_collection
INTERNALERROR>     session.perform_collect()
INTERNALERROR>   File "/Users/cjameson/workspace/cjcjameson/langchain/venv/lib/python3.9/site-packages/_pytest/main.py", line 672, in perform_collect
INTERNALERROR>     hook.pytest_collection_modifyitems(
INTERNALERROR>   File "/Users/cjameson/workspace/cjcjameson/langchain/venv/lib/python3.9/site-packages/pluggy/_hooks.py", line 433, in __call__
INTERNALERROR>     return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)
INTERNALERROR>   File "/Users/cjameson/workspace/cjcjameson/langchain/venv/lib/python3.9/site-packages/pluggy/_manager.py", line 112, in _hookexec
INTERNALERROR>     return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
INTERNALERROR>   File "/Users/cjameson/workspace/cjcjameson/langchain/venv/lib/python3.9/site-packages/pluggy/_callers.py", line 155, in _multicall
INTERNALERROR>     return outcome.get_result()
INTERNALERROR>   File "/Users/cjameson/workspace/cjcjameson/langchain/venv/lib/python3.9/site-packages/pluggy/_result.py", line 108, in get_result
INTERNALERROR>     raise exc.with_traceback(exc.__traceback__)
INTERNALERROR>   File "/Users/cjameson/workspace/cjcjameson/langchain/venv/lib/python3.9/site-packages/pluggy/_callers.py", line 80, in _multicall
INTERNALERROR>     res = hook_impl.function(*args)
INTERNALERROR>   File "/Users/cjameson/workspace/cjcjameson/langchain/libs/langchain/tests/unit_tests/conftest.py", line 43, in pytest_collection_modifyitems
INTERNALERROR>     only_extended = config.getoption("--only-extended") or False
INTERNALERROR>   File "/Users/cjameson/workspace/cjcjameson/langchain/venv/lib/python3.9/site-packages/_pytest/config/__init__.py", line 1622, in getoption
INTERNALERROR>     raise ValueError(f"no option named {name!r}") from e
INTERNALERROR> ValueError: no option named 'only_extended'

====================================================================================================== 4 skipped, 1 warning, 1 error in 3.80s =======================================================================================================
make: *** [coverage] Error 3
```

### Suggestion:

It looks like the `pytest_addoption` in `tests/unit_tests/conftest.py` is not being found.

This stack-overflow attributes it to pytest not being able to find `conftest.py` files in nested directories. https://stackoverflow.com/a/31526934

The recommendations to create a plugin or move the conftest.py files don't seem palatable, but let me know if maybe that's the thing to do

Given the re-organization into `libs/langchain`, that could have messed up pytest local development.

I'm curious if/how it works in CI ...

--- a/libs/langchain/Makefile
+++ b/libs/langchain/Makefile
@@ -7,20 +7,18 @@ all: help
 # TESTING AND COVERAGE
 ######################
 
+# Define a variable for the test file path.
+TEST_FILE ?= tests/unit_tests/
+
 # Run unit tests and generate a coverage report.
 coverage:
 	poetry run pytest --cov \
 		--cov-config=.coveragerc \
 		--cov-report xml \
-		--cov-report term-missing:skip-covered
+		--cov-report term-missing:skip-covered \
+		$(TEST_FILE)
 
-# Define a variable for the test file path.
-TEST_FILE ?= tests/unit_tests/
-
-test:
-	poetry run pytest --disable-socket --allow-unix-socket $(TEST_FILE)
-
-tests: 
+test tests:
 	poetry run pytest --disable-socket --allow-unix-socket $(TEST_FILE)
 
 extended_tests:

